packages/frontend/src/features/catalog/components/CatalogMetadata.tsx

```diff
@@ -53,111 +59,302 @@ export const CatalogMetadata: FC = () => {
-                size="l"
-                weight={700}
-                onDoubleClick={() => {
-                    history.push(
-                        `/projects/${projectUuid}/tables/${metadata.modelName}`,
-                    );
-                }}
-            >
-                {metadata.name}
-            </Text>
-            <ScrollArea
-                variant="primary"
-                className="only-vertical"
-                offsetScrollbars
-                scrollbarSize={8}
-            >
-                <MarkdownPreview
-                    style={{ fontSize: 'small' }}
-                    source={metadata.description}
-                />
+        <Stack h="100vh" spacing="xl">
+            <Group spacing="xs">
+                <Avatar
+                    size="sm"
+                    radius="xl"
+                    styles={(theme) => ({
+                        root: {
+                            border: `1px solid ${theme.colors.gray[1]}`,
+                        },
+                        placeholder: {
+                            color: theme.colors.gray[7],
+                            backgroundColor: 'white',
+                        },
+                    })}
+                >
+                    <MantineIcon icon={IconTable} />
+                </Avatar>
+                <Text
+                    fs="lg"
+                    fw={600}
+                    onDoubleClick={() => {
+                        history.push(
+                            `/projects/${projectUuid}/tables/${metadata.modelName}`,
+                        );
+                    }}
+                >
+                    {metadata.name}
+                </Text>
+            </Group>
 
-                <Tabs defaultValue="overview">
-                    <Tabs.List>
-                        <Tabs.Tab value={'overview'} mx="md">
-                            Overview
-                        </Tabs.Tab>
-                        <Tabs.Tab value={'analytics'} mx="md">
+            <Tabs
+                color="dark"
+                defaultValue="overview"
+                styles={(theme) => ({
+                    tabsList: {
+                        borderBottom: `1px solid ${theme.colors.gray[3]}`,
+                    },
+                    panel: {
+                        paddingTop: theme.spacing.xl,
+                        height: `calc(100vh - 220px)`,
+                        overflowY: 'scroll',
+                    },
+                    tab: {
+                        paddingRight: theme.spacing.xs,
+                        paddingLeft: 0,
+                        fontSize: theme.fontSizes.sm,
+                        fontWeight: 500,
+                        '&[aria-selected="true"]': {
+                            color: theme.colors.gray[9],
+                        },
+                        '&[aria-selected="false"]': {
+                            color: theme.colors.gray[6],
+                        },
```

Comment:
I think it's typically kind of frowned upon to style things based on aria-props. They have some of the most variable browser behavior.

This is probably fine for something not very critical, so leave it if you want. But could we use a class or data-prop instead? I know we arent controlling all this so if mantine wants it this way, so be it.

Reply:
oops! 
used the `data-active` data attribute. Yea, thank you for checking this!! better now that we use a `data-prop` :+1: